TRENZ.Docs.API.Test/PathTraversalTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Security;
+
+namespace TRENZ.Docs.API.Test;
+
+[TestClass]
+[TestCategory("Services")]
+public class PathTraversalTests
+{
+    private const string MacOsRoot = "/var/folders/ab/cdefg/T/TRENZ.Docs.API/trenz-docs-test";
```

Comment:
_Technically_ the only paths the service will ever see are Linux paths... This is really minor, but the correct name would probably be `UnixPathRoot` or something along those lines